lib/the_little_thinkers_space/upload_paths_helper.ex

```diff
@@ -36,4 +36,36 @@ defmodule TheLittleThinkersSpace.UploadPathsHelper do
         {:error, :no_delete_path}
     end
   end
+
+  def thumbnail_path(path) do
+    case is_binary(path) do
+      true ->
+        extension = Path.extname(path)
+        thumbnail_path = String.replace(path, extension, ".jpg")
+        {:ok, thumbnail_path}
+
+      _ ->
+        {:error, :no_thumbnail_path}
+    end
+  end
+
+  def thumbnail_show_path(nil) do
+    {:ok, nil}
+  end
+
+  def thumbnail_show_path(path) do
+    case is_binary(path) do
+      true ->
+        thumbnail_show_path =
+          path
+          |> String.replace("app/lib/the_little_thinkers_space-0.1.0/", "")
+          |> String.replace("priv/static/", "/")
+          |> String.replace("//", "/")
+
+        {:ok, thumbnail_show_path}
+
+      _ ->
+        {:error, :no_thumbnail_show_path}
+    end
```

Comment:
If it is not a binary, and not nil, what else can it be?

Reply:
Is this a trick question? lol I'm going with "nothing" -> I removed this option completely